source/user-manual/ruleset/index.rst

```diff
@@ -1,5 +1,7 @@
 .. Copyright (C) 2021 Wazuh, Inc.
-
+.. meta::
+  :description: This part of the documentation explains how to install, update, and contribute to the Wazuh Ruleset. 
+  
```

Comment:
I can still see the old meta description under the title of this section (lines 9/11) and it needs to be removed.
If you've already fixed it later on, dismiss this comment.